src/main/jbake/content/usingexamples001.adoc

```diff
@@ -36,12 +36,12 @@ software from
 Jakarta EE 8 Software Development Kit
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-GlassFish Server Open Source Edition 5.0 is targeted as the build and
+GlassFish Server 5.1 is targeted as the build and
 runtime environment for the tutorial examples. To build, deploy, and run
 the examples, you need a copy of GlassFish Server and, optionally,
 NetBeans IDE. To obtain GlassFish Server, you must install the Jakarta EE 8
 Software Development Kit (SDK), which you can download from
-`http://www.oracle.com/technetwork/java/javaee/downloads/index.html`.
+`https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip`.
```

Comment:
Better to point at the download page: https://eclipse-ee4j.github.io/glassfish/download.